src/views/EmployeeList.vue

```diff
@@ -34,10 +34,10 @@
 
                                 </div>
                             </td>
-                            <td class="py-2 px-4 border-b text-center">{{ employee.name }}</td>
+                            <td class="py-2 px-4 border-b text-center">{{employee.firstName}}</td>
                             <td class="py-2 px-4 border-b text-center">{{employee.personnelCode}}</td>
-                            <td class="py-2 px-4 border-b text-center">{{employee.phoneNumber}}</td>
-                            <td class="py-2 px-4 border-b text-center">{{employee.state}}</td>
+                            <td class="py-2 px-4 border-b text-center">{{employee.phoneNumbe}}</td>
+                            <td class="py-2 px-4 border-b text-center">{{employee.address}}</td>
```

Comment:
We don't need to see address here

Reply:
Our database and backend information do not contain any information based on state. Because of that, I have changed it to static.